src/lib.rs

```diff
@@ -1,14 +1,34 @@
-use red4ext_rs::prelude::*;
+use red4ext_rs::{prelude::*};
 
 define_plugin! {
     name: "example",
     author: "jekky",
     version: 1:0:0,
     on_register: {
         register_function!("SumInts", sum_ints);
+        register_function!("PluginName", plugin_name);
+        register_function!("Convert", convert);
+        register_function!("Append", append);
     }
 }
 
 fn sum_ints(ints: Vec<i32>) -> i32 {
     ints.iter().sum()
 }
+
+fn plugin_name() -> String {
+    String::from("RED4EXT.RS.EXAMPLE")
+}
+
+fn convert(name: String) -> TweakDBID {
```

Comment:
maybe a more obvious name
```suggestion
fn create_tweakdb_id(name: String) -> TweakDBID {
```

Reply:
addressed in [066f81b](https://github.com/jac3km4/red4ext-rs-example/pull/1/commits/066f81babfc902534393155be6240a6145898e97)